.gitignore

```diff
@@ -0,0 +1 @@
+terraform-provider-buildkite
```

Comment:
I guess this is here to ignore the temporary change you were making to do local dev? I'd suggest not including this here because it's likely to cause confusion for future development.